changelogs/fragments/47247-docker_container-add-runtime-option.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
This should be `minor_changes` and not `bugfixes`, since it is a new feature (and not a bugfix).

Reply:
done